pkgs/games/redeclipse/default.nix

```
@@ -0,0 +1,110 @@
+    { name = "crosshairs"; sha256 = "14w8ysqzdsx9bzpfbl700jzngbh14rdghhjdf6zd6jlkvrl6754r"; }
+    { name = "dziq"; sha256 = "056imqszvp90j7cgz52ly0f31px64gsrmvm9k2c78ldbx87jnhc3"; }
+    { name = "elyvisions"; sha256 = "1bsgr0gr7njydj8fqclh0a27lrsyic3xfd5a4vwggw7g54azpgk2"; }
+    { name = "fonts"; sha256 = "00ibisza1qci0ghf2rynyf28l6r3nqhfzjf80k6gg76q4v7p1myx"; }
+    { name = "freezurbern"; sha256 = "07l9ldk9b82f12c13wcg5xxdf15bw0yjxk3vvk8v3ygrl2mwksyr"; }
+    { name = "john"; sha256 = "1jdmwkrdi5b9pivkm22rxhmkk1db9dx6l54wzcc23cvdz04ij93k"; }
+    { name = "jojo"; sha256 = "0f7kjy43fbk9kw8fip6bbw4gn3pryh0fndlahjfkaysrx98krdj3"; }
+    { name = "jwin"; sha256 = "0nc8dndnpqk2ad6316a8k6kgzsrkpwvk8s4gyh7aqfi4axfclril"; }
+    { name = "luckystrike"; sha256 = "04jiipqahphmvz5cd74dygr62dlvv6l4iglb8hzh4pp8frhls8bq"; }
+    { name = "maps"; sha256 = "0an46ipjvw4h0nxvb6qvnzp1cdkzlkiinqz4zh9lmxy1ds0gclim"; }
+    { name = "mayhem"; sha256 = "15k10imm2wr6c6fq35n4r99k7kz7n9zdnjlw6dmdq6fba67i6sbc"; }
+    { name = "mikeplus64"; sha256 = "0v4wiiivm3829j4phlavy22n4c6k6ib9ixxpdz7r6ysg5cdkaw33"; }
+    { name = "misc"; sha256 = "13rfgwrlfhflz6inbkg3fypyf8im0m49sw112b17qrw2zgp8i1rz"; }
+    { name = "nieb"; sha256 = "0z0h9jdn2gkkjil3vsvwidb1p2k09pi5n3wjxza12hhvqmcs7q8f"; }
+    { name = "nobiax"; sha256 = "08bfp4q6gbfis18bp1h4d0hqssk79jc4fhyjxnv21dbam4v4mnkn"; }
+    { name = "particles"; sha256 = "1vsx3fgg19xggxfhz3vlrh6nqhmw7kl9kmxrvb2j84blp00vd6z2"; }
+    { name = "philipk"; sha256 = "14irscg80607i5k5l2ci0n9nwibvda2f3xsykgv96d0vldrp5n5a"; }
+    { name = "projectiles"; sha256 = "09bnfsrywirwgjm6d7ff5nicx6w6b7w9568x9pb5q0ncaps3l85s"; }
+    { name = "props"; sha256 = "1dlabbprlkif8af3daf9nbgcwgxiymvj0yiphqhlri8ylfy2vpz4"; }
+    { name = "skyboxes"; sha256 = "14bi3md5y47cvb9ybipdvksz40gqsqw2r0lh3zzqb4acq367w18y"; }
+    { name = "snipergoth"; sha256 = "0m8rvvy5n8n9pm0b5cqvzsxsw51mqk8m7s1h3qc849b38isliwq2"; }
+    { name = "sounds"; sha256 = "0ivf3w5bphz5pzzx6kwcb67vbly1l19cgv3s0cyp8n87afiqj5rd"; }
+    { name = "textures"; sha256 = "0qdmgx7zbcqnb9rrga2izr93p5inirczhddfxs504rsnv0v8vyxm"; }
+    { name = "torley"; sha256 = "05ppyhghq859cbbxzj3dnl9fcx3ghy04ds1pylypwg2hsxzbjwcd"; }
+    { name = "trak"; sha256 = "0g3vq86q91a3syli38lwc8ca4ychfwsmmqf85kqzfzyd627ybclm"; }
+    { name = "ulukai"; sha256 = "0asa5fz400impklcg6dy2f7jiaqfc1sn1c36fpg8jd01gw66lw93"; }
+    { name = "unnamed"; sha256 = "0rz5683j7sfwkcycfypbv4b0ihp0qwn9rzskfsabwc1s5g324917"; }
+    { name = "vanities"; sha256 = "13f18783rc8cjf22p61zr8m5g1migzlx05fzl8xnbjdkqq4cdyix"; }
+    { name = "vegetation"; sha256 = "1y5d97nfmvax7y4fr0y5v0c8zb1ajkqwx76kjd4qc9n4spdsi5sc"; }
+    { name = "weapons"; sha256 = "103g1dhxv5ffz4ddg2xcbshbgv9606chsbas3pzk6h9ybqsyjrqh"; }
+    { name = "wicked"; sha256 = "1884rk34a2dj83gz82rc4zh3ch0dyj5221hvsr0a5h60578i7yj6"; }
+  ];
+in
+
+stdenv.mkDerivation rec {
+  name = "${baseName}-${version}";
```

```suggestion
  pname = "redeclipse";
```

is enough - you can get rid of `basename`.

---

Resolved with 4bb447076f0.